sources/PreferenceInfo.h

```diff
@@ -12,6 +12,7 @@ typedef NS_ENUM(NSInteger, PreferenceInfoType) {
     kPreferenceInfoTypeCheckbox,
     kPreferenceInfoTypeInvertedCheckbox,  // true=checked, false=unchecked. Handy when inverting a checkbox's text, but the user defaults key can't be changed.
     kPreferenceInfoTypeIntegerTextField,
+    kPreferenceInfoTypeFloatTextField,
```

Comment:
Gah, what a mess. Please call the new one double, and some day I'll need to find all the instances of Float and rename them to Double.